OmniKit/PumpManager/PodCommsSession.swift

```diff
@@ -28,6 +28,11 @@ public enum PodCommsError: Error {
     case podSuspended
     case podFault(fault: PodInfoFaultEvent)
     case commsError(error: Error)
+    case podChange
+    case activationTimeExceeded
+    case rssiTooLow
+    case rssiTooHigh
+    case debugFault(str: String)
```

Comment:
remove testing code

Reply:
Quick note. Having debugFault isn't just for debugging the pairing code itsself (especially now that there will be RSSI limit enforcement that some users might want to tweak), but also for developers needing to be able to test RileyLinks and RileyLink analogues.